src/applications/vaos/services/location/transformers.js

```diff
@@ -192,6 +192,14 @@ export function transformFacility(facility) {
         system: 'phone',
         value: facility.phone?.main,
       },
+      {
```

Comment:
I think it would be better if this object didn't exist at all in the telecom array when there's no covid phone, but that's pretty nitpicky, and it requires some moving around of the regular phone code. I don't think we have to change it, just something to think about.